src/components/atoms/Accordion/AccordionUL.mdx

```diff
@@ -0,0 +1,18 @@
+---
+name: AccordionUL
+---
+
+import AccordionUL from './AccordionUL';
+import { Playground, Props } from 'docz';
+
+# AccordionUL
+
+## Properties
+
+<Props of={AccordionUL} />
+
+## Basic Usage
+
+<Playground>
+  <AccordionUL accordionFill={accordionFill} index={i} />
```

Comment:
Where is `accordionFill` coming from? I don't see it defined, so guessing this would break Docz from running